templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,13 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 61**
+
+* [`GET /events`](/api/get-events): Added new `Admin only` policy
+  to `private_message_policy` event.
+* [`POST /register`](/api/register-queue): Added new `Admin only`
+  policy to `realm_private_message_policy` event.
+
```

Comment:
Here `private_message_policy` or `realm_private_message_policy` is not an event. And realm_private_message_policy is not returned as a part of any event, it is included in `page_params` object. You can read other points in `changelog.md` to get idea about how to improve this.